### DISCUSSION TITLE
20231221 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by cgbosse** [nautilus-cyberneering/.github#74] [Meetings]
Time: 8:30 GMT+0  Canary Islands Time (9:30 in Vaduz)

Location:
<https://meeting.hodl10.io/ConsistentClassroomsMergeSideways>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - x
- JCelano(@josecelano) - x
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by mario-nt** [comment]
### Other

- Set up new PC
- Read https://torrust.com/live-demo-beta-v3

### GUI

- Refactored https://github.com/torrust/torrust-index-gui/pull/297

**Comment by josecelano** [comment]
### Torrust Compose

- PR: [New configuration for running the app in a Digital Ocean Droplet](https://github.com/torrust/torrust-compose/pull/4).

### Torrust Demo

Live demo deployed to a Digital Ocean Droplet:

- http://index.torrust-demo.com/
- https://tracker.torrust-demo.com/announce
- udp://tracker.torrust-demo.com:6969/announce

### Torrust Website

New blog post: [Deploying Torrust To Production](https://torrust.com/deploying-torrust-to-production)

Shared on:

- https://www.reddit.com/r/trackers/comments/18i8e09/torrust_index_and_tracker_live_demo/?utm_source=share&utm_medium=web2x&context=3 (10.5k views)
- https://www.linkedin.com/posts/josecelano_announcing-the-beta-v300-live-demo-torrust-activity-7141409118333915137-7rgI?utm_source=share&utm_medium=member_desktop (144 impressions)

### Torrust Index

- PR: [fixed issue about too verbose SQL statements logs](https://github.com/torrust/torrust-index/pull/416).

### Next

- Fix small issues in Tracker, Index and Index GUI repos. For example:
  -  [Tracker logs are too verbose ](https://github.com/torrust/torrust-tracker/issues/539).
  - ...
- Update blog articles (docker configuration changed and repos and env var names):
  - https://github.com/torrust/torrust-website/issues/121
  - https://github.com/torrust/torrust-website/issues/114
- Write draft BEPs:
  -  https://github.com/torrust/teps/issues/12
  - ...

### Discussions

- Persist tracker statistics? They are reset when you restart the tracker. Se https://github.com/torrust/torrust-tracker/discussions/543
- Full private mode for the Index? See https://www.reddit.com/r/trackers/comments/18i8e09/comment/kdda2qj/?utm_source=share&utm_medium=web2x&context=3

**Reply by josecelano** [reply]
I've created a [new discussion](https://github.com/torrust/torrust-tracker/discussions/545) for the "full private mode".

**Reply by josecelano** [reply]
And a [new discussion to change linting rules for merging PRs](https://github.com/torrust/torrust-tracker/discussions/546).

**Comment by grmbyrn** [comment]
### Torrust Website
* Apply recommended changes to `formatTime` function in `date.ts`
* Create responsive hamburger menu for use on smaller screens
* Create modal for top of the screens to let user know about the latest blog post with a link to it
* Make blog posts wider on larger screens
* Research newsletter options. Create newsletter on [Mailchimp](http://mailchimp.com/) and add embedded form for users to sign up to newsletter through website
* Look into options for giving users the option to share blog posts to their social media

### Secure Git Guide
* Fix error in dark mode

### Revision
* Create flashcards on [Brainscape](https://www.brainscape.com/) for revision of important topics such as GitHub, SvelteKit, etc.

### Other
* Set up new laptop
